packages/react/src/reactrouterv6.tsx

```diff
@@ -203,9 +205,9 @@ export function withSentryReactRouterV6Routing<P extends Record<string, any>, R
     const navigationType = _useNavigationType();
 
     _useEffect(() => {
-      // Performance concern:
-      // This is repeated when <Routes /> is rendered.
-      routes = _createRoutesFromChildren(props.children);
```

Comment:
It's actually repeated internally by React Router 6. So caching doesn't fix it. But I don't think there is anything we can do anyways 🤷

Reply:
Removed!